src/api/patients/patient-details-controller.js

```diff
@@ -42,11 +38,20 @@ export const patientDetailsController = async (req, res) => {
     const responseBody = {
       coreMessageUrl,
       fragmentMessageIds,
+      // TODO: remove this duplicated `conversationIdFromEhrIn` field,
+      // after updating ehr-out to use the field name "inboundConversationId"
```

Comment:
Could you please put the Ticket number on this TODO, so we know what it refers to?

Reply:
Sure! Just added ticket number to TODO